packages/xo-web/src/common/intl/messages.js

```diff
@@ -44,6 +44,7 @@ const messages = {
   // ----- Copiable component -----
   copyToClipboard: 'Copy to clipboard',
   copyUuid: 'Copy {uuid}',
+  clickToCopyUuid: 'Click to copy {uuid}',
```

Comment:
I think we already have something like that.

Reply:
no, I didn't find it.

Reply:
https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/intl/messages.js#L47